### DISCUSSION TITLE
How to Uninstall MagiskOnWSALocal

### DISCUSSION THREAD

**Discussion by ninuola** [LSPosed/MagiskOnWSALocal#239] [Q&A] [answered]
Hi, please is there is a way to completely uninstall MagiskOnWSALocal from my Windows 11 Machine?
I will like to do a clean-up and then reinstall the stock WSA.

Many thanks.

**Comment by Howard20181** [comment, marked as answer]
Right-click `Windows Subsystem for Android Settings` in the Start menu and select Uninstall
